### ISSUE TITLE
seed => jax.random.PRNGKey in constructor/function arguments

### ISSUE THREAD

**Issue by leonard-gleyzer** [leonard-gleyzer/connex#7]
This agrees more with JAX/Equinox style.
